invenio_rdm_records/config.py

```diff
@@ -559,6 +559,19 @@ def lock_edit_published_files(service, identity, record=None, draft=None):
 RDM_RECORDS_MAX_MEDIA_FILES_COUNT = 100
 """Max amount of media files allowed to upload in the deposit form."""
 
+RDM_MEDIA_FILES_DEFAULT_QUOTA_SIZE = 10**10
+"""Default size for a bucket in bytes for media files."""
+
+RDM_MEDIA_FILES_DEFAULT_MAX_FILE_SIZE = RDM_MEDIA_FILES_DEFAULT_QUOTA_SIZE
+"""Default maximum file size for a bucket in bytes for media files."""
+
+RDM_FILES_DEFAULT_QUOTA_SIZE = None
```

Comment:
Let's add a comment here explaining the backwards compatibility and why we set it to none here